config/triplea/schemas/triplea_maps.json

```diff
@@ -0,0 +1,46 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "Maps set",
+  "description": "A set of maps",
+  "type": "array",
+  "items": {
+    "title": "Map",
+    "description": "A map",
+    "type": "object",
+    "properties": {
+      "mapName": {
+        "description": "The map name",
+        "type": "string"
+      },
+      "mapCategory": {
```

Comment:
This property is optional because the code currently doesn't support defining a category for skins and tools.  If not defined, the code assumes `EXPERIMENTAL`.